.golangci.yaml

```diff
@@ -9,10 +9,14 @@ run:
 
 # Settings of specific linters
 linters-settings:
+  gocritic:
+    enabled-checks:
+      - dupImport
```

Comment:
It seems that we don't enable this check in k/k. So, I'm hesitant that we enable it:  https://github.com/kubernetes/kubernetes/blob/12217672a3511f5e461ecb87b3e22422ab30bdbf/hack/golangci.yaml#L203-L206

@alculquicondor @mimowo WDYT?

Reply:
I suspect that it wasn't added just because there are a lot of violations and fixing all of them now is just toil on the reviewers :)